pkg/controllers/cluster/cluster_controller.go

```diff
@@ -402,7 +402,7 @@ func (c *Controller) monitorClusterHealth(ctx context.Context) (err error) {
 					c.clusterHealthMap.delete(clusterName)
 					return true, nil
 				}
-				klog.Errorf("Getting a cluster to retry updating cluster health error: %v", clusterName, err)
+				klog.Errorf("Getting a cluster %v to retry updating cluster health error: %v", clusterName, err)
```

Comment:
@Garrybest should we optimize this log to make it easier to understand? Any comments?

By the way, I think this log level could be `Warning`, `klog.Errorf` --> `klog.Warningf`.

Reply:
I think we can delete this line. The err will return and afterwards the details will be printed.

`klog.Errorf("Update health of Cluster '%v' from Controller error: %v. Skipping.", cluster.Name, err)`

Reply:
I delete this log.